Utilities/File.cpp

```diff
@@ -813,6 +813,8 @@ fs::file::file(const std::string& path, bs_t<open_mode> mode)
 
 	class windows_file final : public file_base, public get_native_handle
 	{
+		const uint IO_CHUNK_SIZE = 1024 * 1024 * 8;  // 8 MiB
```

Comment:
Surely typing in a single literal is better than performing 2 multiplications?

Reply:
static const or constexpr maybe?

Reply:
Const value will be evaluated at compile time so no runtime implication.

Reply:
@PZerua Yeah neither of those are valid C++ because of what @HaronK said.

Reply:
No, this is class body, so const uint should occupy some memory in the object. It must be ```static constexpr```

Reply:
@Nekotekina Hm, `windows_file` is a local class though so it can't be `static`, just `constexpr`. Should I just put the declaration just above (outside) the class?

Reply:
Then it can be just a line above outside the class. And don't use ALL_CAPS style.